[2022-05-23 20:15:11,253] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-07T00:00:00+00:00 [queued]>
[2022-05-23 20:15:11,285] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-07T00:00:00+00:00 [queued]>
[2022-05-23 20:15:11,285] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-05-23 20:15:11,285] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2022-05-23 20:15:11,285] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-05-23 20:15:11,352] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): second_task> on 2021-11-07 00:00:00+00:00
[2022-05-23 20:15:11,372] {standard_task_runner.py:52} INFO - Started process 1643 to run task
[2022-05-23 20:15:11,376] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'second_task', 'scheduled__2021-11-07T00:00:00+00:00', '--job-id', '312', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp9vfpirsp', '--error-file', '/tmp/tmpx01s2d6a']
[2022-05-23 20:15:11,378] {standard_task_runner.py:80} INFO - Job 312: Subtask second_task
[2022-05-23 20:15:11,567] {task_command.py:369} INFO - Running <TaskInstance: our_first_dag_v5.second_task scheduled__2021-11-07T00:00:00+00:00 [running]> on host 061a475e6c14
[2022-05-23 20:15:11,758] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=gerson
AIRFLOW_CTX_DAG_ID=our_first_dag_v5
AIRFLOW_CTX_TASK_ID=second_task
AIRFLOW_CTX_EXECUTION_DATE=2021-11-07T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-07T00:00:00+00:00
[2022-05-23 20:15:11,759] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-23 20:15:11,759] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2022-05-23 20:15:11,786] {subprocess.py:85} INFO - Output:
[2022-05-23 20:15:11,788] {subprocess.py:92} INFO - hey, I am task2 and will be running after task1!
[2022-05-23 20:15:11,789] {subprocess.py:96} INFO - Command exited with return code 0
[2022-05-23 20:15:11,923] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=second_task, execution_date=20211107T000000, start_date=20220523T201511, end_date=20220523T201511
[2022-05-23 20:15:11,999] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-23 20:15:12,067] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
